src/app/vault/vault.component.ts

```diff
@@ -93,6 +95,10 @@ export class VaultComponent implements OnInit, OnDestroy {
 
             this.showProviders = (await this.userService.getAllProviders()).length > 0;
 
+            // They have an available sponsorship and they have not redeemed it yet
```

Comment:
I think we can remove the comment, the property name `showRedeemSponsorship` is more helpful than the comment.